apps/frontend-pwa/src/components/practiceQuiz/ElementStack.tsx

```
@@ -237,6 +238,37 @@ function ElementStack({
+                        criterionAcc[criterion.criterionId] = criterion.response
+                        return criterionAcc
+                      }, {})
+                      return itemAcc
+                    }, {})
+                    return caseAcc
+                  },
+                  {}
+                )
+
+              return {
+                ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)